test/api/v01/request_helper.rb

```diff
@@ -43,6 +44,9 @@ def wait_status_csv(job_id, status, options)
       get "0.1/vrp/jobs/#{job_id}", options
 
       assert_equal 200, last_response.status, last_response.body
+      unless last_response.body.is_a?(String)
+        assert_includes ['queued', 'working', 'completed'], JSON.parse(last_response.body)['job']['status']
+      end
 
       break if last_response.content_type == 'text/csv;'
```

Comment:
would this work? I think if the content is not `text/csv` then we should be able to parse the response without any issue
```suggestion

      break if last_response.content_type == 'text/csv;'

      assert_includes ['queued', 'working', 'completed'], JSON.parse(last_response.body)['job']['status']
```